crates/ruff_linter/src/rules/pycodestyle/rules/blank_lines.rs

```diff
@@ -424,11 +427,12 @@ impl<'a> Iterator for LinePreprocessor<'a> {
         // Number of consecutive blank lines directly preceding this logical line.
         let mut blank_lines = BlankLines::Zero;
         let mut first_logical_line_token: Option<(LogicalLineKind, TextRange)> = None;
-        let mut last_token: TokenKind = TokenKind::EndOfFile;
+        let mut last_token = TokenKind::EndOfFile;
         let mut parens = 0u32;
 
-        while let Some((token, range)) = self.tokens.next() {
-            if matches!(token, TokenKind::Indent | TokenKind::Dedent) {
+        while let Some(token) = self.tokens.next() {
```

Comment:
Are the fields on Token public? Could we instead write
```suggestion
        while let Some(Token { kind, range }) = self.tokens.next() {
```

Reply:
No, currently they're private. I guess it's fine to make it public? Probably not in this PR though